minvws/nl-kat-coordination
# Fix nmap-ports regex pattern not allowing 80

Allow any decimal with 1-4 digits instead of up to 5000, 500, 30 or 5 (for decimals having the same amount of digits as these numbers)

### Changes

This handles the case where we want to allow arbitrary decimals up to 10.000. The complicated statements are built such that the max is `65535`, but we were not handling this specific case for larger than 5000, 500, 30 or 5. Of course I used 2, 20, 200 and 2000 in my initial unit tests...

### Issue link

Closes https://github.com/minvws/nl-kat-coordination/issues/3639

### Demo


### QA notes

Please check that you can add the sensible settings for the nmap-ports boefje as described in the issue!

---

### Code Checklist


---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
